src/ser/mod.rs

```diff
@@ -176,10 +203,10 @@ fn default_depth_limit() -> usize {
 }
 
 fn default_new_line() -> String {
-    #[cfg(not(target_os = "windows"))]
+    // #[cfg(any(test, not(target_os = "windows")))]
```

Comment:
why are these changes needed?

Reply:
Some of the tests expect linux line endings, so this was just a temporary fix to get them to pass. Didn't mean to push it 😄 Those tests should probably be fixed, though that's not relevant to the PR

Reply:
Oh, looks like I did make it handle tests at build time actually. Would you like to implement this another way or just use the `any(test, ...` approach?